modeladminutils/edit_handlers.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.template.loader import render_to_string
+from django.utils.safestring import mark_safe
+
+from wagtail.wagtailadmin.edit_handlers import BaseChooserPanel
+
+from modeladminutils.widgets import GenericModelChooser
+
+
+class BaseGenericModelChooserPanel(BaseChooserPanel):
```

Comment:
Do we need separate panel and cant just use http://docs.wagtail.io/en/v1.7/topics/snippets.html for our models ?